modules/oadp-release-notes-1-2-0.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-2-0_{context}"]
+= OADP 1.2.0 release notes
+
+The OADP 1.2.0 release notes include new features and lists of fixed bugs and known issues.
+
+[id="new-features_{context}"]
+== New features
+
+.Resource timeouts
+The new `resourceTimeout` option specifies how many minutes to wait for several Velero resources before timeout. This applies for such resources as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availablity. 10 minutes is the default setting.
```

Comment:
```suggestion
The new `resourceTimeout` option specifies how many minutes to wait for several Velero resources before timeout. This option applies to resources such as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availability. Ten minutes is the default setting.
```

- use words if a number occurs at the beginning of a sentence. IBM Styleguide
- `This` was unclear. Use `This option applies ...` or `It applies ...`

Reply:
Just one additional nit here. The first sentence sounds a bit incomplete to me. My suggestion (including Gaurav's suggestion above):

```suggestion
The new `resourceTimeout` option specifies the timeout duration in minutes for waiting on various Velero resources. This option applies to resources such as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availability. The default duration is ten minutes.
```